src/main/java/beam/agentsim/events/handling/BeamEventsLogger.java

```diff
@@ -153,8 +171,10 @@ public void setEventsFileFormats() {
 
     public Map<String,String> getAttributes(Event event) {
         Map<String,String> attributes = event.getAttributes();
-        if(getLoggingLevel(event) == LoggerLevels.SHORT && eventFieldsToAddWhenVerbose.containsKey(event.getClass())){
-            // remove fields from attributes heres
+        if(getLoggingLevel(event) == LoggerLevels.SHORT && eventFieldsToDropWhenShort.containsKey(event.getClass())){
+            for (Map.Entry<Class, String> entry : eventFieldsToDropWhenShort.entrySet()) {
```

Comment:
This is incorrect and will drop all fields across all classes that are keys in eventFieldsToDropWhenShort. Note see above that eventFieldsToDropWhenShort should be a multimap.